### PR TITLE
Testing remote running tests in parallel

### PR DESCRIPTION
**Pull request by Marketionist**
**This pull request adds remote running tests in parallel and some additional configs**

### CODE REVIEW

**Comment by Marketionist** [comment]
@Xotabu4 please review

**Comment by Marketionist** [comment]
@Xotabu4 please review updates

File: protractor.conf.js [hunk 1]
```diff
@@ -14,20 +29,52 @@ module.exports.config = {
         customTimeout: 3000,
         customMinTimeout: 500,
     },
-     /**
-     * How long to wait for a page to load.
-     */
+
+    // The timeout in milliseconds for each script run on the browser.
+    allScriptsTimeout: 60000,
```
Comment (Xotabu4) [inline protractor.conf.js]: Reduce timeout to 20-30 secs - Waiting for Angular - http://www.protractortest.org/#/timeouts

**Review by Xotabu4 — APPROVED** [review]
LGTM, can merge. 
But i suggesting to reduce timeout of allScriptsTimeout

**Pull request merged**